R/AbstractGraphReporter.R

```diff
@@ -94,11 +94,21 @@ AbstractGraphReporter <- R6::R6Class(
             )
 
             # Round the double columns to three digits for formatting reasons
-            numCols <- names(which(unlist(lapply(tableObj$x$data, is.double))))
+
+            # what are the double columns (but still may contain values with all integers)?
+
```

Comment:
could you please remove the blank lines between comments and code? So e.g. change this to

```
# some comment
the_code()
```

Reply:
Sure working on it now